packages/app-builder-backend/src/ddb.ts

```diff
@@ -0,0 +1,138 @@
+import {
+  DynamoDBClient,
+  DescribeTableCommand,
+  CreateTableCommand,
+  QueryCommand,
+  PutItemCommand,
+  AttributeValue,
+} from '@aws-sdk/client-dynamodb'
+
+import { App } from './entities/app'
+import { Page } from './entities/page'
+
+const { APPS_TABLE_NAME = 'apps', GSI_NAME = 'userId-index', DYNAMODB_ENDPOINT, AWS_REGION = 'eu-west-2' } = process.env
+
+export const ddb = new DynamoDBClient({
+  endpoint: DYNAMODB_ENDPOINT,
+  region: AWS_REGION,
+})
+
+const getCreateTableCommand = (tableName: string): CreateTableCommand => {
```

Comment:
Can this be done with CDK?

Reply:
this is only used locally, am using CDK in dev/prod